### PR TITLE
[ ADD ] Support for react-native

### PR DESCRIPTION
**Pull request by grovertb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "css-select": "~2.1.0",
     "dom-serializer": "~1.1.0",
     "entities": "~2.0.2",
+    "events": "^3.2.0",
```
Comment (fb55) [inline package.json:31]: `events` isn't used anywhere in this module. Let's add this to the module actually pulling in the dependency, or have users add it to their own `package.json`s.

I know that `htmlparser2` currently depends on it, but that will be fixed before the next cheerio release.

**Review by fb55 — COMMENTED** [review]
Checking if `Buffer` exists is a good idea. Happy to merge if we can remove the `events` change.

File: package.json (hunk 1 above)
Comment (grovertb) [reply package.json]: I just removed the package "events"

**Comment by fb55** [comment]
Thanks a lot @grovertb!

**Pull request merged**